From 9b34b2428127e39c2b2a7f219e20432d3a6c291a Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Fri, 5 Dec 2025 15:07:19 +0100
Subject: [PATCH 944/954] fix(stm32mp1): use GIC in BL2

Right now, GIC is unusable as DEVICE2 is not mapped in BL2.
Fix this.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 3b0fd300bf3738cf40392f4cb7793be0e7ddf5ef fix(stm32mp1): use GIC in BL2]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I5ed9be77610d166af2834000b83b2937d4bb802b
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/515704
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 plat/st/stm32mp1/stm32mp1_private.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/plat/st/stm32mp1/stm32mp1_private.c b/plat/st/stm32mp1/stm32mp1_private.c
index c003878233..800c0e8d48 100644
--- a/plat/st/stm32mp1/stm32mp1_private.c
+++ b/plat/st/stm32mp1/stm32mp1_private.c
@@ -71,9 +71,7 @@ static const mmap_region_t stm32mp1_mmap[] = {
 	MAP_SRAM_ALL,
 #endif
 	MAP_DEVICE1,
-#if STM32MP_RAW_NAND
 	MAP_DEVICE2,
-#endif
 	{0}
 };
 #endif
-- 
2.53.0

